src/gui/optionsdlg.cpp

```diff
@@ -165,83 +165,88 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
 
     // Connect signals / slots
-    connect(m_ui->comboProxyType, SIGNAL(currentIndexChanged(int)), this, SLOT(enableProxy(int)));
-    connect(m_ui->checkRandomPort, SIGNAL(toggled(bool)), m_ui->spinPort, SLOT(setDisabled(bool)));
+    // shortcuts for frequently used signals and slots
```

Comment:
And please modify appropriate comment to describe why exactly you use these shortcuts.